src/main/resources/static/js/script.js

```diff
@@ -152,17 +151,26 @@ async function shoot(x, type) {
     console.log(shootMessage);
     switch (shotResult) {
         case 'MISS': {
+            shotMastSound.load()
+            shotWaterSound.load()
+            shotWaterSound.play()
             await shootCell(cellId, 'cellHit')
             swapBoards()
             activateCells()
             break
         }
         case 'SHIP_HIT': {
+            shotMastSound.load()
+            shotWaterSound.load()
+            shotMastSound.play()
             await shootCell(cellId, 'shipHit')
             activateCells()
             break
         }
         case 'SHIP_SUNK': {
+            shotMastSound.load()
```

Comment:
i tu

Reply:
Jeżeli poprzedni strzał był trafiony, to odpaliło to shotMastSound. Jeżeli odpowiedź na kolejny trafiony strzał zdążyła przyjść zanim poprzedni shotMastSound przestał grać, to do tego kolejnego trafionego strzału nie będzie już grać muzyka. Na pewno usunąć?